config/rbac/service_account.yaml

```diff
@@ -1,4 +0,0 @@
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: sriov-network-config-daemon
```

Comment:
I have zero context on your bundle contents, but had a quick question - This ServiceAccount manifest metadata.Name wouldn't match the `serviceAccountName` specified in the CSV manifest, so deleting this may not be needed. It's possible I'm missing something and this was renamed to `sriov-network-config-daemon_service_account.yaml` like I'm seeing in the kustomize.yaml changes but I didn't see that change in this PR.

Reply:
I found we have `sriov-network-config-daemon_service_account.yaml` with identical content. So I remove this file and modify the kustomization file.  This is just a cleanup.

Reply:
Thanks for confirming that - this lgtm.